lib/filterx/expr-regexp.c

```diff
@@ -28,6 +28,14 @@
 #include "filterx/object-list-interface.h"
 #include "filterx/object-dict-interface.h"
 #include "compat/pcre.h"
+#include "scratch-buffers.h"
+
+#define FILTERX_FUNC_REGEXP_SUBST_USAGE "regexp_subst(string, pattern, replacement, [" \
```

Comment:
To me `[]` implies that these are passed in an array, I think we should remove the brackets, the rest is good.